tedana/tests/data/cornell_three_echo_outputs.txt

```diff
@@ -1,6 +1,20 @@
-adaptive_mask.nii.gz
-betas_OC.nii.gz
-betas_hik_OC.nii.gz
+S0map.nii.gz
+T2starmap.nii.gz
+desc-ICAAcceptedZ_components.nii.gz
+desc-ICAAccepted_components.nii.gz
+desc-ICAZ_components.nii.gz
```

Comment:
These could possibly all be replaced with `fitlins`-style outputs, in which statistical maps have the `_statmap` suffix and the `stat` entity can tell us if the map is a z-score, parameter estimate, etc. map. The problem is that BIDS-Models (and fitlins' outputs) are not supported in the actual specification at this point. I don't know how in-flux they are either.

Reply:
So this one would be `desc-ICA_stat-z_components.nii.gz` in this case, I think?

Reply:
can you mix the `components` suffix with the `stat` key in fitlins ? I'm less up to date on BEP002, but can review it

Reply:
I don't think the file formatting is actually in BEP002. I think it's just about the jsons specifying the models. It was a mistake to mention BIDS-Models, since it just seems to be in `fitlins`.

Reply:
Oh but I'm also not sure if `stat` can be combined with `components`. I feel like it should be allowed though. It's a logical combination...